### PR TITLE
[DO NOT MERGE] CQ-4312433 CQ-4312432 Add sample MS Dynamics and Salesforce Form Data Model and Data Source

### PR DESCRIPTION
**Pull request by Meet001**
Adding sample MS Dynamics and Salesforce Form Data Model and Data Source

Adding sample MS Dynamics and Salesforce Form Data Model and Data Source in content. To reduce configuration time required for forms with fdm users.

## Motivation and Context

Adding sample MS Dynamics and Salesforce Form Data Model and Data Source

## How Has This Been Tested?

Yes, Tested content on local sdk and on skyline server pushed via vsts.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by Meet001** [comment]
@rismehta @sakshi-arora1 please review

File: src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/.content.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <jcr:content
+        cq:lastModified="{Date}2020-12-21T22:03:43.978+05:30"
+        cq:lastModifiedBy="admin"
+        jcr:lastModified="{Date}2020-12-21T22:03:43.915+05:30"
+        jcr:lastModifiedBy="admin"
+        jcr:primaryType="cq:PageContent"
+        jcr:title="Salesforce Cloud Service"
+        sling:resourceType="fd/fdm/gui/components/admin/fdmcloudservice/rest"
+        access_token_uri="https://login.salesforce.com/services/oauth2/token"
+        authenticationType="OAuth 2.0"
```
Comment (sakshi-arora1) [inline src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/.content.xml:13]: @Meet001 Some one would need to manually do oAuth for this to work, right? So how are we proposing this as reference?

Comment (Meet001) [reply src/main/archetype/ui.content/src/main/content/jcr_root/conf/__appId__/settings/cloudconfigs/fdm/salesforce-cloud-service/.content.xml:13]: @sakshi-arora1 Yes Customer would need to fill their MS Dynamics/Salesforce instance details and connect to oauth in properties page. this is similar to the way it was done for MS Dynamics in 6.5 https://experienceleague.adobe.com/docs/experience-manager-65/forms/form-data-model/ms-dynamics-odata-configuration.html?lang=en#configure-cloud-service-for-your-microsoft-dynamics-service

**Comment by rismehta** [comment]
LGTM !

**Review by sakshi-arora1 — APPROVED** [review]

**Comment by sakshi-arora1** [comment]
@Meet001 Lets hold this checkin till testing infra is finalised. @rismehta will be sharing that soon.

**Pull request closed without merging**